core/web/replay_controller.go

```diff
@@ -25,10 +25,10 @@ func (bdc *ReplayController) ReplayFromBlock(c *gin.Context) {
 	}
 
 	// check if force_broadcast query string parameter provided
-	var forceBroadcast bool
+	var force bool
 	var err error
-	if fb := c.Query("force_broadcast"); fb != "" {
-		forceBroadcast, err = strconv.ParseBool(fb)
+	if fb := c.Query("force"); fb != "" {
+		force, err = strconv.ParseBool(fb)
 		if err != nil {
 			jsonAPIError(c, http.StatusUnprocessableEntity, errors.Wrap(err, "boolean value required for 'force_broadcast' query string param"))
```

Comment:
Should be `force` here too?

Reply:
Yep missed that, put up a PR: https://github.com/smartcontractkit/chainlink/pull/6399/files

it shouldn't affect functionality though just the error message

Reply:
Fixed on develop and added here :cherries: